debug/Dockerfile

```diff
@@ -24,3 +24,14 @@ RUN --mount=type=ssh sh -c \
 
 # Set default shell to Zsh
 RUN sed -i 's|root:x:0:0:root:/root:/bin/ash|root:x:0:0:root:/root:/bin/zsh|' /etc/passwd
+
+# Install AWS CLI
+RUN apk add --no-cache \
+        python3 \
+        py3-pip \
+    && pip3 install --upgrade pip \
+    && pip3 install \
+        awscli \
+    && rm -rf /var/cache/apk/*
+
+RUN aws --version
```

Comment:
Leave this guy out as it's for your own debugging purposes.

Reply:
got it